modules/effects/spec/effects_metadata.spec.ts

```diff
@@ -11,17 +12,24 @@ describe('Effects metadata', () => {
         @Effect({ dispatch: false })
         c: any;
         d = createEffect(() => of({ type: 'a' }), { dispatch: false });
+        @Effect({ dispatch: false, resubscribeOnError: false })
+        e: any;
         z: any;
       }
 
       const mock = new Fixture();
+      const expected: EffectMetadata<Fixture>[] = [
+        { propertyName: 'a', dispatch: true, resubscribeOnError: true },
+        { propertyName: 'c', dispatch: false, resubscribeOnError: true },
+        { propertyName: 'b', dispatch: true, resubscribeOnError: true },
+        { propertyName: 'd', dispatch: false, resubscribeOnError: true },
+        { propertyName: 'e', dispatch: false, resubscribeOnError: false },
+      ];
 
-      expect(getSourceMetadata(mock)).toEqual([
-        { propertyName: 'a', dispatch: true },
-        { propertyName: 'c', dispatch: false },
-        { propertyName: 'b', dispatch: true },
-        { propertyName: 'd', dispatch: false },
-      ]);
+      expect(getSourceMetadata(mock)).toEqual(
+        jasmine.arrayContaining(expected)
```

Comment:
any reason why this has `jasmine.arrayContaining`?

Reply:
I'd the order of the elements is unpredictable (or at least not intuitive to me), so I'd rather check the content and then check the length to match as well.